### PR TITLE
Publish Python generated protobuf code to PyPI

### PR DESCRIPTION
**Pull request by jtbandes**
**Public-Facing Changes**
Published a new package to PyPI called `foxglove-schemas-protobuf`. Import classes as follows:

```py
from foxglove_schemas_protobuf.RawImage_pb2 import RawImage
```

### CODE REVIEW

**Comment by defunctzombie** [comment]
👍

File: python/Makefile [hunk 1]
```diff
@@ -12,9 +18,9 @@ generate-protobuf:
 		../schemas/proto/foxglove/*.proto
 	mv foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove/* foxglove-schemas-protobuf/foxglove_schemas_protobuf
 	rmdir foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove
-	sed -E -i '' 's/from foxglove import/from . import/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.py
-	sed -E -i '' 's/import foxglove\.(.+)$$/from . import \1 as foxglove_\1/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.pyi
-	sed -E -i '' 's/foxglove\./foxglove_/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.pyi
+	sed -E $(SED_ARGS) 's/from foxglove import/from . import/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.py
```
Comment (james-rms) [inline python/Makefile:21]: might be a little more bulletproof to write a python script for this?

Comment (jtbandes) [reply python/Makefile:21]: Seems a bit hard 😕 https://stackoverflow.com/a/4427835/23649

Comment (jtbandes) [reply python/Makefile:21]: I'm working on it but it feels not better. Looks kind of like this so far

```python
def main():
    subprocess.check_call(
        [
            "protoc",
            "--python_out=foxglove-schemas-protobuf/foxglove_schemas_protobuf",
            "--mypy_out=foxglove-schemas-protobuf/foxglove_schemas_protobuf",
            "--proto_path=../schemas/proto",
            *glob.glob("../schemas/proto/foxglove/*.proto"),
        ]
    )

    for f in glob.glob(
        "foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove/*"
    ):
        shutil.move(f, "foxglove-schemas-protobuf/foxglove_schemas_protobuf")
    shutil.rmtree("foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove")

    for f in glob.glob("foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.py"):
```

File: python/.flake8 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[flake8]
```
Comment (james-rms) [inline python/.flake8:1]: does it make sense to run linters for zero lines of foxglove-written python?

Comment (jtbandes) [reply python/.flake8:1]: A fair question. I guess this sets us up for it if we ever want to add code here. But I suppose we’d likely add any supporting code in a separate package, not this package.

**Pull request merged**